INSTALL.md

```diff
@@ -62,6 +62,9 @@ The GalSim package also requires
   The latter is preferred, since this is now where all future development of 
   this package is happening.
 
+* The future module, which is used to ease compatibility between Python 2
+  and Python 3.
```

Comment:
I think you mean permitted, not required.  But yes, good point.  I'll go through and check the language around those statements.